README.md

```diff
@@ -1,12 +1,16 @@
 # Todo-App
 
+## CI Builds
+
+To view the application CI builds in Travis-CI visit: [https://travis-ci.com/github/Noah-Vincenz/DevOps-Course-Starter/builds](https://travis-ci.com/github/Noah-Vincenz/DevOps-Course-Starter/builds).
+
 ## Initial One-time Setup
 
 To setup the application run 
 ```bash
 $ vagrant up
```

Comment:
Not a change here, but it seems a bit overkill to tell a new dev that even if they intend to use docker for example, they should configure vagrant just to copy over the .env template file!